auraloss/perceptual.py

```diff
@@ -55,14 +55,17 @@ class FIRFilter(torch.nn.Module):
     a sampling rate of 44.1 kHz, considering adjusting this value at differnt sampling rates.
     """
 
-    def __init__(self, filter_type="hp", coef=0.85, fs=44100, ntaps=101, plot=False):
+    def __init__(self, filter_type="hp", coef=0.85, fs=44100, ntaps=101, butter_order=2, butter_freq=(250, 5000), butter_filter_type="bandpass", plot=False):
```

Comment:
I think adding the butterworth parameters to this constructor make the behavior a bit confusing since we also have the `filter_type` parameter which will get ignored. I do not know the best solution yet, but it feels like we should instead just pass `taps` into this constructor. Then we will need to supply some helper functions for hp and butterworth filters that will produce a tensor of taps. What do you think?